<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="open_fileB"/>
   </widget>
   <addaction name="menuOpciones"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_fileB">
   <property name="text">
    <string>Abrir Archivo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
